python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -302,55 +303,56 @@ def trace(model: nn.Module, input_sample=None, accelerator=None):
                              model is a LightningModule with any dataloader attached.
         :param accelerator: The accelerator to use, defaults to None meaning staying in Pytorch
                             backend. Only 'openvino' is supported for now.
-        :return: A PytorchOpenVINOModel using openvino for inference if accelerator='openvino'
+        :return: Model with different acceleration(OpenVINO/ONNX).
```

Comment:
may be we can change all `ONNX` to`ONNX Runtime`

Reply:
or maybe leave it to me, I will change the function str.

Reply:
Will be good if we get unite name in a ORT PR. thx.